_search-plugins/concurrent-segment-search.md

```diff
@@ -138,7 +138,9 @@ The `search.concurrent.max_slice_count` setting can take the following valid val
 
 ## The `terminate_after` search parameter
 
-The [`terminate_after` search parameter]({{site.url}}{{site.baseurl}}/api-reference/search/#url-parameters) is used to terminate a search request once a specified number of documents has been collected. In the non-concurrent search workflow, this count is evaluated for each shard. However, in the concurrent search workflow, it is evaluated for each leaf slice instead in order to avoid synchronizing document counts between threads. With concurrent search, the request performs more work than expected because each segment slice on the shard collects up to the specified number of documents. The intent to terminate collection after the threshold is reached is evaluated at the slice level. Thus, the hit count in the results will be greater than the `terminate_after` threshold but less than `slice_count * terminate_after`. The actual number of returned hits will be controlled by the `size` parameter.
+The [`terminate_after` search parameter]({{site.url}}{{site.baseurl}}/api-reference/search/#url-parameters) is used to terminate a search request once a specified number of documents has been collected. If you include the `terminate_after` parameter in a request, concurrent segment search is disabled and the request is run in a non-concurrent manner.
```

Comment:
```suggestion
The `terminate_after` [search parameter]({{site.url}}{{site.baseurl}}/api-reference/search/#url-parameters) allows you to terminate a search request after a specified number of documents has been collected. When you include the `terminate_after` parameter in a request, concurrent segment search is disabled and the request runs in a non-concurrent manner.
```

Reply:
The name of the parameter is included in the link because that's what the section is called.

Reply:
My bad, the section is URL parameters but we want the user to look for a specific one, so having the link on the whole name reinforces that.